src/js/node/dns.js

```diff
@@ -1,6 +1,7 @@
 // Hardcoded module "node:dns"
 // only resolve4, resolve, lookup, resolve6, resolveSrv, and reverse are implemented.
 const dns = Bun.dns;
+const util = require("node:util");
```

Comment:
Instead of loading all of `node:util`, we can choose to instead load only the symbol we need:

```suggestion
const utilPromisifyCustomSymbol = Symbol.for("nodejs.util.promisify.custom")
```

Reply:
Hmm...
It turns out, there's a pair of symbols: `nodejs.util.promisify.custom` and `util.promisify.custom`

Reply:
`// TODO: register shared symbol promisify.custom` — it's time.